src/hooks/useSeoPost.ts

```diff
@@ -45,7 +45,7 @@ export const useSeoPost = (post: PostPageData): void => {
         name: author.name,
         url: generatePath(PATHS.AUTHOR, { authorUsername: author.username, lang: i18n.language }),
       })),
-      images: [coverUrl],
+      image: [coverUrl],
```

Comment:
hum le typage n'est pas bon ici non ? image est de type `ImageObject` ou `URL` et là c'est un tableau avec une string

Reply:
le problème devrait être résolu... enfin, j'espère ;)

Reply:
Array of strings works as well it seems: https://developers.google.com/search/docs/appearance/structured-data/article